msal-client-credential-certificate/README.md

```diff
@@ -0,0 +1,238 @@
+---
+services: active-directory
+platforms: Java
+endpoint: Microsoft identity platform
+page_type: sample
+author: ramya25
+level: 200
+client: Java console daemon app
+service: Microsoft Graph
+languages:
+  - java
+products:
+  - azure
+  - azure-active-directory
+  - java
+  - office-ms-graph
+description: "This sample demonstrates how a daemon console app can get an access token to call Microsoft Graph using MSAL4J."
```

Comment:
We should state that this sample uses a certificate while the other uses a client secret.